build.gradle

```diff
@@ -1,19 +1,21 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        gradlePluginPortal()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:4.1.2'
```

Comment:
Do we need to do this upgrade at the same time?  Seems like it is probably unrelated to publishing?

Reply:
I think so. Without updating the build tools version, the `gradle-maven-publish-plugin` breaks with an error:
```
Caused by: org.gradle.api.UnknownDomainObjectException: SoftwareComponentInternal with name 'release' not found.
```

Since @barnhill will be creating another PR to update the dependencies, I opted to update this to the latest that matches the stable version for Android Studio.